web/src/click_handlers.js

```diff
@@ -924,4 +924,11 @@ export function initialize() {
     $(".settings-header.mobile .fa-chevron-left").on("click", () => {
         settings_panel_menu.mobile_deactivate_section();
     });
+    // Workaround for issue #26941, which allows for element outside settings overlay
```

Comment:
Probably better to just document what this is doing; no need to mention the issue here, I don't think.

Reply:
Alright @karlstolley . Thank you for the review. I've updated a detailed explanation of what it's doing and removed the issue code.